### PR TITLE
Team Hackbrighters Github Universe 2016 Conference blog post

### PR DESCRIPTION
**Pull request by arbonap**
A summary of mine and @go-bears's wonderful experience at Github Universe - 2016.

### CODE REVIEW

**Comment by anasofiapinho** [comment]
Hey @arbonap! I was looking at the blog post and it is lovely! <3

There are some issues that we need you to fix before we merge the PR:
- Some images are a bit pixelated, especially 1, 3 and 4
- The images don't have credits. Can you add them?

Thank you! :)

**Comment by anasofiapinho** [comment]
Thank you @arbonap and @go-bears! 

@alicetragedy please take a look to see if it's ready to merge or if they need to make any changes.

**Review by alicetragedy — APPROVED** [review]
Thank for taking over the review @anasofiapinho! It's totally good from my side and ready to go.

**Pull request merged**
